packages/angular/tsconfig.json

```diff
@@ -17,6 +17,7 @@
     "lib": ["dom", "es2019", "dom.iterable"]
   },
   "angularCompilerOptions": {
+    "compilationMode": "partial",
```

Comment:
Using "partial" compilation mode is [recommended](https://angular.io/guide/creating-libraries#publishing-libraries) from Angular:

> Contains portable code that can be consumed by Ivy applications built with any version of Angular from v12 onwards.